README.md

```diff
@@ -28,8 +28,14 @@ Our [`python-plugwise`](https://github.com/plugwise/python-plugwise) python modu
 
 # Changelog
 
-## NEW FEB 2022 [0.21.1]
-- Smile: fix issue #238
```

Comment:
The topic-part of this line should not be deleted from the 0.21.1 release info!

Reply:
Wrong edit